src/test/java/dataloader/CsvLoaderTest.java

```diff
@@ -97,7 +97,7 @@ public void testBuildNumericSampleFromCsvWithExistingColumn()throws IOException{
         Map<String, List<String>> dataSet = CsvDataLoader.MapLoader.parseFile(file);
         assertNotNull(dataSet);
 
-        NumericSample sample = CsvDataLoader.MapLoader.buildSample(dataSet, "X");
+        NumericSample sample = CsvDataLoader.MapLoader.buildNumericSample(dataSet, "X");
         assertEquals(sample.size(), 25);
     }
```

Comment:
I am writing here because I cannot above: 
the `junit.framework.Assert` is deprecated. please replace with `org.junit.Assert`

Reply:
should be fixed in next commit